src/plugins/expressions/common/execution/execution.ts

```diff
@@ -91,6 +98,20 @@ export class Execution<
    */
   private readonly abortController = new AbortController();
 
+  /**
+   * Promise that rejects if/when abort controller sends "abort" signal.
+   */
+  private readonly abortRejection = new Promise<never>((resolve, reject) => {
```

Comment:
Can this be replaced with https://github.com/elastic/kibana/blob/b54388746dedfadd8f5432f9005c063271afc129/src/plugins/data/common/utils/abort_utils.ts#L39?

Reply:
Done.